paper-tooltip.html

```diff
@@ -13,6 +13,7 @@
 <link rel="import" href="../neon-animation/neon-animation-runner-behavior.html">
 <link rel="import" href="../neon-animation/animations/fade-in-animation.html">
 <link rel="import" href="../neon-animation/animations/fade-out-animation.html">
+<link rel="import" href="../neon-animation/web-animations.html">
```

Comment:
No. That's the web animations polyfill. It used to get imported by each animation: https://github.com/PolymerElements/neon-animation/pull/206/files#diff-909aade6cee6a6b071fa1791f1ff3cbeL13, which was bad because Chrome, for example doesn't need it.
Now it's the responsibility of the user to do that; example: https://github.com/PolymerElements/neon-animation/pull/206/files#diff-82c4006fe2f8b0b62134f4ce2a40e1b0R21